src/gei/IBlobServiceClientFactory.cs

```diff
@@ -0,0 +1,9 @@
+using Azure.Storage.Blobs;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter
+{
+    public interface IBlobServiceClientFactory
+    {
+        public BlobServiceClient Create(string connectionString);
+    }
```

Comment:
As I already mentioned, we could get rid of this factory if we expect the connection string to be present at the composition time.

Reply:
Passing in the connection string as an arg is preferred